tw-tasks-management/src/main/java/com/transferwise/tasks/management/TasksManagementPortController.java

```diff
@@ -184,6 +188,8 @@ public ResponseEntity<GetTasksStuckResponse> getTasksStuck(@RequestBody(required
     return callWithAuthentication(() -> {
       ITasksManagementService.GetTasksStuckResponse serviceResponse = tasksManagementService
           .getTasksStuck(new ITasksManagementService.GetTasksStuckRequest()
+              .setTaskType(request != null ? request.getTaskType() : null)
+              .setTaskSubType(request != null ? request.getTaskSubType() : null)
```

Comment:
Is this just one sub type then?

Reply:
one task will have one type and one sub type; across all tasks of the same type there might be many sub types.
idea here would be to allow filtering by only one type and one subtype, but might make sense to allow a list for both params actually

Reply:
In my own approach before, I had planned on making the java API use a var-arg for types - as I had a use case where I wanted to get all tasks of a particular type

Reply:
(basically doing a fork-join on 2 task types)